Vagrantfile

```diff
@@ -55,7 +69,7 @@ Vagrant.configure("2") do |config|
   if $image_version != "current"
       config.vm.box_version = $image_version
   end
-  config.vm.box_url = "https://storage.googleapis.com/%s.release.core-os.net/amd64-usr/%s/coreos_production_vagrant.json" % [$update_channel, $image_version]
+  config.vm.box_url = "https://storage.googleapis.com/%s.release.core-os.net/amd64-usr/%s/coreos_production_vagrant_virtualbox.json" % [$update_channel, $image_version]
```

Comment:
I'm gonna keep a copy of the old Vagrantfile at `Vagrantfile.user-data` for now. Just in case someone wants to use virtualbox vagrant with a non-alpha release.